tests/compat.sh

```diff
@@ -126,7 +126,7 @@ print_usage() {
     printf "            \tAlso available: GnuTLS (needs v3.2.15 or higher)\n"
     printf "  -M|--memcheck\tCheck memory leaks and errors.\n"
     printf "  -v|--verbose\tSet verbose output.\n"
-    printf "     --list-test-case\tList all potential test cases (No Execution)\n"
+    printf "  -l|--list-test-cases\tList all potential test cases (No Execution)\n"
```

Comment:
Preexisting: `-l` is not actually a recognized option, and I don't think it needs to be: this isn't something we commonly do from the command line. Please remove it from the help text here, and also from the options of `ssl-opt.sh`.

Reply:
Done